packages/rum-core/src/common/apm-server.js

```diff
@@ -207,25 +207,32 @@ class ApmServer {
   }
 
   ndjsonErrors(errors) {
-    return errors.map(function(error) {
-      return NDJSON.stringify({ error })
-    })
+    return errors.map(error => NDJSON.stringify({ error }))
+  }
+
+  ndjsonMetricsets(metricsets) {
+    const timestamp = microtime()
```

Comment:
The real issue with creating the timestamp here is that the clients time is sometimes inaccurate (sometimes it's off in years). We have seen this before, even if there are a few clients that have a wrong time it will clearly show up in the data since the APM server would generate indices based on date. Therefore, the users will see indices with dates that are clearly wrong (e.g. in the future) and managing these indices quickly gets out of hand.

Reply:
I think this is still a blocker for the break down feature.